app/models/compute_resources/foreman/model/ovirt.rb

```diff
@@ -186,6 +186,10 @@ def destroy_vm(uuid)
       true
     end
 
+    def supports_vms_pagination?
+      true
```

Comment:
As far as I understood, this change is depending onto abenari/rbovirt#102 and a new rbovirt release with that change in. rbovirt 0.1.x, however, depends on rest-client > 1.7.0. If there's the possibility of a rbovirt 0.0.x release with the required change in, that wouldn't require the rest-client update, of course.